VMEncryption/main/handle.py

```diff
@@ -745,29 +753,30 @@ def enable_encryption():
                       code=str(CommonVariables.unknown_error),
                       message=message)
 
+
 def enable_encryption_format(passphrase, disk_format_query, disk_util, force=False):
     logger.log('enable_encryption_format')
     logger.log('disk format query is {0}'.format(disk_format_query))
 
     json_parsed = json.loads(disk_format_query)
 
     if type(json_parsed) is dict:
-        encryption_format_items = [json_parsed,]
+        encryption_format_items = [json_parsed, ]
```

Comment:
Is this proper syntax?  I know it was from before but it looks odd..

Reply:
Did some testing and it does not cause a syntax error, although I'm not sure it is needed either (may have been left over from some prior change).